lean_packages/manifest.json

```diff
@@ -1,28 +1,6 @@
 {"version": 2,
  "packages":
- [{"url": "https://github.com/mhuisi/lean4-cli",
```

Comment:
Eek, what happened here? I nuked the `lean_packages` directory, assuming nothing was checked in there, because I wanted to get to the most recent version of `Std`.